student-info-system/scheduler/scheduler.py

```diff
@@ -0,0 +1,21 @@
+from django.utils import timezone
+
+import pytz
+from apscheduler.schedulers.background import BackgroundScheduler
+
+from sis.models import Student, Message, Profile
+
+
+def academic_probation_check():
```

Comment:
I'd probably move this into Student as a class method. That way the specifics are less coupled to the scheduler.

Truly, we'd have a business layer where things like this go. But no.

Reply:
No longer relevant